gnucash/gnome-utils/gnc-gnome-utils.c

```diff
@@ -129,6 +130,7 @@ gnc_options_dialog_set_new_book_option_values (GNCOptionDB *odb)
                         num_source_is_split_action);
     }
 }
+#endif
```

Comment:
To be removed still? Or kept around in case it turns out to be needed here anyway?

Reply:
Removed/